examples/settings.py

```diff
@@ -36,5 +36,5 @@
 PORT = 443
 SECURE = True
 VERSION = "2018-10-01"
-HOST = "platform.exabyte.io"
+HOST = "platform.mat3ra.com"
```

Comment:
Maybe let's use LFS for the image

Reply:
Done via [DOC: add an image for README as an lfs object](https://github.com/Exabyte-io/api-examples/pull/70/commits/876bc745ef57d255a1f80a039c743f6d546d498d).